### PR TITLE
Feature/documentation/geoprocessing architecture

### PR DESCRIPTION
**Pull request by aagm**
### Overview

This pr adds:
* Inittial documentation on backend architecture and data management strategies.
* Initial setup of the geoprocessing DB
* Adds new functionality to the marxan executer notebook that allow us modify in place the files before executing marxan

### Testing instructions

`make notebooks` to test the notebook functionality  
`make start` to init the initial setup of the Geoprocessing DB

## Checklist before submitting

- [x] Meaningful commits and code rebased on `develop`.
- [ ] If this PR adds feature that should be tested for regressions when
      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
- [ ] Update CHANGELOG file

### CODE REVIEW

File: docs/README_science_and_data.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* Validation
+* Geometry sanitization (Repair and simplification)
+* Spatial intersection
+* Area calculation
+* Split by property
+* Stratification; special spatial intersection
+
+## Vector tile service
+
+* Serve vector tiles from the db on the fly with a Redis cache upfront
```
Comment (hotzevzl) [inline docs/README_science_and_data.md:49]: not something to finalise here, but I wonder if it may make sense to use a CDN for this

**Review by hotzevzl — COMMENTED** [review]
code/configs and documentation/diagrams in this PR look ok to me

I suggest perhaps to try to link the geoprocessing operations listed in `README_science_and_data.md` to the geoprocessing architecture diagram, and maybe to add some actions/arrows on the diagram itself to clarify the intended order of steps: right now I am not clear about the intended workflow (but I suppose this is a WIP, so it's ok if you plan to update this later of course).

In terms of notebooks, I got stuck on a compilation error during the build of one of the images - if you would like me to test this and give more detailed feedback please let me know and I'll reach out for help on resolving the build issue.

File: docs/README_science_and_data.md (hunk 1 above)
Comment (aagm) [reply docs/README_science_and_data.md:50]: probably yes. But we can reach that point when we attack the service.

**Comment by aagm** [comment]
Ok added the references in the data document, as you can see there is still a lot to work on define, but one step after another. Merging this

**Pull request merged**